Adyen.EcommLibrary.Test/BaseTest.cs

```diff
@@ -1,22 +1,23 @@
 ﻿using Adyen.EcommLibrary.Constants;
-using Adyen.EcommLibrary.HttpClient;
-using Adyen.EcommLibrary.HttpClient.Interfaces;
-using Adyen.EcommLibrary.Model;

```

Comment:
how does Model.Amount work as you do not include Model ?

Reply:
Model.Amount is the part of the namespace Adyen.EcommLibrary.Model. It is separated from Adyen.EcommLibrary.Model.Nexo.Amount